dlls/wscript

```diff
@@ -16,7 +16,7 @@ def configure(conf):
 		hlDefNode = conf.path.find_resource("./hl.def")
 
 		if hlDefNode is not None:
-			conf.env.append_unique('LINKFLAGS', f'/def:{hlDefNode.abspath()}')
+			conf.env.append_unique('LINKFLAGS', '/def:{}'.format(hlDefNode.abspath()))
```

Comment:
Let's then use % notation, i.e. `'/def:%s' % hlDefNode.abspath()`, it's shorter and easier to read, IMO.

Reply:
As you wish, I'm not a huge fan of C style formatting in Python :)

Reply:
Fan or not fan, but even python programmers on my main job uses this style.